kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
Like I said, for security reason you want a value. 128m was fine. By putting it to 0 you remove any value, so anybody not using the plugins or when APIs don’t have the plugin enabled, they are vulnerable.

So yes the only solution is to put a value, and document that any value in set when adding the plugin on an API cannot exceed the configured value.

Ideally, if possible, try to catch when a value is set higher than the max value. 

> On 04 Jun 2015, at 01:59, shashi ranjan <notifications@github.com> wrote:
> 
> In kong.yml <https://github.com/Mashape/kong/pull/292#discussion_r31683093>:
> 
> > @@ -90,7 +91,7 @@ nginx: |
> >      real_ip_recursive on;
> >  
> >      # Other Settings
> > -    client_max_body_size 128m;
> > +    client_max_body_size 0m;
> Ok its 1 mb. But if we set a max payload size, plugin would be useless beyond that size limit. I thought whole idea for this pluingin is to give control to user. We do have to mention in our doc that by default this field is disabled.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Mashape/kong/pull/292/files#r31683093>.
>